repository/t.json

```diff
@@ -2451,7 +2462,7 @@
 			"details": "https://github.com/civAnimal/tinyscheme_autocomplete",
 			"releases": [
 				{
-					"sublime_text": "*",
+					"sublime_text": ">=4169",
```

Comment:
why are you changing this other package as well?

Reply:
Any changes to this other package was a mistake.